varats/varats/projects/cpp_projects/hyteg.py

```diff
@@ -0,0 +1,110 @@
+"""Adds the HyTeg framework as a project to VaRA-TS."""
+import typing as tp
+
+import benchbuild as bb
+from benchbuild.command import WorkloadSet, Command, SourceRoot
+from benchbuild.utils.cmd import make, cmake, mkdir
+from benchbuild.utils.revision_ranges import SingleRevision
+from benchbuild.utils.settings import get_number_of_jobs
+from plumbum import local
+
+from varats.experiment.workload_util import WorkloadCategory, RSBinary
+from varats.paper.paper_config import PaperConfigSpecificGit
+from varats.project.project_domain import ProjectDomains
+from varats.project.project_util import (
+    get_local_project_git_path,
+    BinaryType,
+    ProjectBinaryWrapper,
+)
+from varats.project.sources import FeatureSource
+from varats.project.varats_command import VCommand
+from varats.project.varats_project import VProject
+from varats.utils.git_util import ShortCommitHash, RevisionBinaryMap
+from varats.utils.settings import bb_cfg
+
+
+class HyTeg(VProject):
+    """
+    C++ framework for large scale high performance finite element simulations
+    based on (but not limited to) matrix-free geometric multigrid.
+
+    Note:
+        Currently HyTeg CANNOT be compiled with the Phasar passes activated
+        in vara.
+        Trying to do so will crash the compiler
+
+        If you use Dune with an experiment that uses the vara compiler,
+        add `-mllvm --vara-disable-phasar` to the projects `cflags` to
+        disable phasar passes.
+        This will still allow to analyse compile-time variability.
+    """
+    NAME = 'HyTeg'
+    GROUP = 'cpp_projects'
+    DOMAIN = ProjectDomains.HPC
+
+    SOURCE = [
+        PaperConfigSpecificGit(
+            project_name="HyTeg",
+            remote="git@github.com:se-sic/hyteg-VaRA.git",
```

Comment:
```suggestion
            remote="https://github.com/se-sic/hyteg-VaRA.git",
```
So this can be used without credentials